middleware/src/util.go

```diff
@@ -39,8 +39,17 @@ func (middleware *Middleware) mountFlashdrive() rpcmessages.ErrorResponse {
 		}
 	}
 
-	// `bbb-cmd.sh flashdrive check` echos only the flashdrive name, if no error occurs
-	flashDriveName := outCheck[0]
+	if len(outCheck) > 1 { // `bbb-cmd.sh flashdrive check` returns only the flashdrive name, if no errors occur
+		log.Printf("Warning: The `bbb-cmd.sh flashdrive check` command returned more than one line. Using '%s' as flashdrive name.\n.", outCheck[len(outCheck)-1])
+	} else if len(outCheck) == 0 {
+		return rpcmessages.ErrorResponse{ // throw an unexpected error if the script does not return anything
+			Success: false,
+			Message: "A unexpected Error occurred: The `bbb-cmd.sh flashdrive check` call did not contain a flashdrive name.",
```

Comment:
Wording nits:
- **An** unexpected Error occurred
- The **output of** `bbb-cmd.sh flashdrive check` call did not contain a flashdrive name.

Reply:
Thanks!